spiders/rei.py

```diff
@@ -499,6 +489,6 @@ def get_product_review(self) -> List:
                         'thumbs_up': review_thumbs_up,
                         'thumbs_down': review_thumbs_down
                     }
-                reviews.append(review)
-        return reviews
+                product_reviews.append(review)      
```

Comment:
move this loop to `parse_bazaarvoice_reviews`. This function should return all reviews in a list